### DISCUSSION TITLE
docs/3.x/developer/app-store/apps/adyen

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#969] [General]
# docs/3.x/developer/app-store/apps/adyen

Overview

https://docs.saleor.io/docs/3.x/developer/app-store/apps/adyen

**Comment by lkostrowski** [comment]
We should describe better Adyen credentials / api token generation, its not that easy

**Reply by lkostrowski** [reply]
Confusing part: Adyen displays key, but it doesnt work yet. First key must be saved. This should be mentioned

**Comment by lkostrowski** [comment]
Webhooks config in app is not documented. No mention in docs, wizard also doesn't give any help

**Comment by lkostrowski** [comment]
Missing docs for manual webhook creation

**Comment by Hanskrogh** [comment]
Github repo missing?
